README.md

```diff
@@ -2,26 +2,21 @@
 
 # Project Name
 
-> Description the project.
-
+> Hello Microverse
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- HTML and CSS
 
 ## Live Demo (if available)
 
 [Live Demo Link](https://livedemo.com)
```

Comment:
Please add a live demo for your project or remove this section for now because when I click on it takes me to the wrong demo.